pycon-fr-2019/A-Dive-Into-Brain-Waves.json

```diff
@@ -0,0 +1,22 @@
+{
+    "title": "A Dive Into Brain Waves",
+    "description": "Advances in cognitive neuroscience, brain imaging technologies and use of sensors which capture physical processes taking place within the brain have given us the ability to interact with the human brain and understand it better. BCI aims at capturing and analysis of electric signals due to neuron interaction in the brain corresponding with some activity. This talk aims at providing an Introduction to Brain-Computer Interface with the basics of the biological aspects concerned with the BCI, and how to leverage python-MNE for processing, analysis and classification of brain waves. The aim of the talk is not to provide end to end information but to just create a curiosity amongst the audience and introduce neuroscience as a field for development. Walk-through of the talk is as follows:\nA) Intro to BCI:\n     1) How are Brain Waves generated\n     2) How are Brain Waves captured\n     3) A small video of fluctuation in brain waves on eye blinking\n     4) Relevance of capturing brain waves from different parts of the brain\n\nB) Present Applications and Research questions associated with BCI\n    1) What already has been done in BCI (history)\n    2) What people are aiming to do now.\n    3) Problems people are facing in data analysis\n\nC) Introducing Python - MNE\n    1) Getting Started with Python-MNE\n    2) Tools Available for Brain Wave analysis.\n    3) Simple Demo of using Python-MNE\n        (i) Simple reading raw EEG data.\n        (ii) Converting the time domain data to frequency for power analysis \n        (iii) Clustering and Classification of brain waves using simple statistical features.",
+    "duration": 3467,
+    "language": "fra",
+    "recorded": "2019-11-02",
+    "speakers": [
+        "Utkarsh Shukla"
+    ],
+    "videos": [
+        {
+            "type": "mp4",
+            "size": 1957845188,
+            "url": "https://dl.afpy.org/pycon-fr-19/Robin%20Huart%20-%20Tests%20-%20la%20jungle%20des%20frameworks/Robin%20Huart%20-%20Tests%20-%20la%20jungle%20des%20frameworks.mp4"
```

Comment:
It looks like this video doesn't match the title/description.

Reply:
Yes I know there's a few of them, I matched between ponyconf and files using levenstein distance between titles, and I spotted it mis-matching a few of them (it typically don't work well when one of the side has been truncated, truncation being a "big edit" from the Levenstein point of view).

Reply:
I'll proofread the thing, consider it WIP. It still miss the youtube videos also (do you have the links to youtube uploads?)